ui/controls.js

```diff
@@ -522,6 +522,29 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     return this.seekBar_ ? this.seekBar_.getValue() : this.video_.currentTime;
   }
 
+  /**
+   * Checking the efficiency of a video
+   * @return {?number}
+   * @export
+   */
+  getBufferFillPercentage() {
```

Comment:
How about `getBufferFullness()`, and make it a fraction from 0 to 1 instead of 0 to 100?

Reply:
Oh, and I also _just_ realized this is in the UI controls.  Please move it to `shaka.Player` instead.  These sorts of calculations should be in the main library API, not a UI helper function.

Reply:
> Oh, and I also just realized this is in the UI controls. Please move it to shaka.Player instead. These sorts of calculations should be in the main library API, not a UI helper function.

I was very confused about where to add this so I added it to UI. I believed members will point this out during the review.
Thank you/